velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -1921,50 +1922,36 @@ template <typename T>
 struct ParseDurationFunction {
   VELOX_DEFINE_FUNCTION_TYPES(T);
 
-  std::unique_ptr<RE2> durationRegex_;
-
-  FOLLY_ALWAYS_INLINE void initialize(
-      const std::vector<TypePtr>& /*inputTypes*/,
-      const core::QueryConfig& /*config*/,
-      const arg_type<Varchar>* /*amountUnit*/) {
-    durationRegex_ =
-        std::make_unique<RE2>(R"(^\s*(\d+(?:\.\d+)?)\s*([a-zA-Z]+)\s*$)");
-  }
-
   FOLLY_ALWAYS_INLINE void call(
       out_type<IntervalDayTime>& result,
       const arg_type<Varchar>& amountUnit) {
+    static const LazyRE2 kDurationRegex{
+        R"(^\s*(\d+(?:\.\d+)?)\s*([a-zA-Z]+)\s*$)"};
+    // TODO: Remove re2::StringPiece != std::string_view hacks.
+    // It's needed because system is old, update please.
```

Comment:
Could you be more specific either here in the the PR comments about what "the system" is, are you referring to CI?